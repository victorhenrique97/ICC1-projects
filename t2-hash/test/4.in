5
10
45
52
29
47
77
22
89
86
63
27
