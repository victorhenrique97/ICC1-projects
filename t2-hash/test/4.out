0: 45
1: 86
2: 52 47 77 22 27
3: 63
4: 29 89
